docs/source/tracking.rst

```diff
@@ -750,6 +835,26 @@ location to server's artifact store. This will be used as artifact location for
 experiments that do not specify one. Once you create an experiment, ``--default-artifact-root``
 is no longer relevant to that experiment.
 
+Starting a server with the ``--serve-artifacts`` option defined enables proxied access for artifacts.
+The uri ``mlflow-artifacts:/`` replaces an otherwise explicit object store destination (e.g., "s3:/my_bucket/mlartifacts")
+for interfacing with artifacts. The client can access artifacts via HTTP requests to the MLflow Tracking Server.
+This simplifies access requirements for users of the MLflow client, eliminating the need to
+configure access tokens or username and password environment variables for the underlying object store when writing or retrieving artifacts.
+
+Provided an Mlflow server configuraton where the ``--default-artifact-root`` is ``s3://my-root-bucket``,
+the following patterns will all resolve to the configured proxied object store location of ``s3://my-root-bucket/mlartifacts``:
+
+ * ``https://<host>:<port>/mlartifacts``
+ * ``http://<host>/mlartifacts``
+ * ``mlflow-artifacts://<host>/mlartifacts``
+ * ``mlflow-artifacts://<host>:<port>/mlartifacts``
+ * ``mlflow-artifacts:/mlartifacts``
```

Comment:
Can we clarify that, when the host is absent from the `mlflow-artifacts` URI, the client assumes that the host is the same as the host component of the tracking URI?